tests/everflow/test_everflow_ipv6.py

```diff
@@ -0,0 +1,592 @@
+            sport=12008,
+            dport=12009
+        )
+
+        self._send_and_check_mirror_packets(setup_info,
+                                            setup_mirror_session,
+                                            ptfadapter,
+                                            duthost,
+                                            test_packet)
+
+    def test_both_subnets(self, setup_info, setup_mirror_session, ptfadapter, duthost):
+        """Verify that we can match packets with both source and destination subnets."""
+        test_packet = self._base_tcp_packet(
+            ptfadapter,
+            setup_info,
+            src_ip="ffbe:0225:7c6b:c000:0000:0000:0000:0010",
+            dst_ip="ffbe:0225:7c6b:d000:0000:0000:0000:0010",
+            sport=12010,
+            dport=12011
+        )
+
+        self._send_and_check_mirror_packets(setup_info,
+                                            setup_mirror_session,
+                                            ptfadapter,
+                                            duthost,
+                                            test_packet)
+
+    def test_fuzzy_subnets(self, setup_info, setup_mirror_session, ptfadapter, duthost):
+        """Verify that we can match packets with non-standard subnet sizes."""
+        test_packet = self._base_tcp_packet(
+            ptfadapter,
+            setup_info,
+            src_ip="ffbe:0225:7c6b:e000:0000:0000:0000:0010",
+            dst_ip="ffbe:0225:7c6b:f000:0000:0000:0000:0010",
+            sport=12012,
+            dport=12013
+        )
+
+        self._send_and_check_mirror_packets(setup_info,
+                                            setup_mirror_session,
+                                            ptfadapter,
+                                            duthost,
+                                            test_packet)
+
+    def _send_and_check_mirror_packets(self, setup, mirror_session, ptfadapter, duthost, mirror_packet):
+        expected_mirror_packet = self._get_expected_mirror_packet(mirror_session, setup, duthost, mirror_packet)
+
+        ptfadapter.dataplane.flush()
+        testutils.send(ptfadapter, self._get_src_port(setup), mirror_packet)
+        _, received_packet = testutils.verify_packet_any_port(
+            ptfadapter,
+            expected_mirror_packet,
+            ports=[self._get_monitor_port(setup, mirror_session, duthost)]
+        )
+
+        logging.info("received: %s", packet.Ether(received_packet).summary())
+
+        inner_packet = self._extract_mirror_payload(received_packet, len(mirror_packet))
+        logging.info("inner_packet: %s", inner_packet.summary())
+        logging.info("expected_packet: %s", mirror_packet.summary())
+        pytest_assert(Mask(inner_packet).pkt_match(mirror_packet), "Mirror payload does not match received packet")
+
+    def _get_src_port(self, setup):
+        return setup["port_index_map"][random.choice(setup["port_index_map"].keys())]
```

Comment:
there is difference between v4 and v6. 
For v4 we are using setup_info based on tor/spine and run test vase for both port as tor and spine.
Looks like here we execute test case only once.

Reply:
We're not testing the forwarding behavior for the mirror session in this test case, so I left the tor/spine part off. Is there another reason to try both?

Reply:
yeah then it is fine.